custom/icds_reports/migrations/sql_templates/database_functions/aggregate_awc_data.sql

```diff
@@ -111,14 +111,17 @@ BEGIN
     'cases_child_health = ut.cases_child_health, ' ||
     'cases_child_health_all = ut.cases_child_health_all, ' ||
     'wer_weighed = ut.wer_weighed, ' ||
-    'wer_eligible = ut.wer_eligible ' ||
+    'wer_eligible = ut.wer_eligible, ' ||
+    'cases_person_beneficiary_v2 = ut.cases_child_health, ' ||
+    'cases_person_has_aadhar_v2 = ut.child_has_aadhar ' ||
   'FROM (SELECT ' ||
     'awc_id, ' ||
     'month, ' ||
     'sum(valid_in_month) AS cases_child_health, ' ||
     'sum(valid_all_registered_in_month) AS cases_child_health_all, ' ||
     'sum(nutrition_status_weighed) AS wer_weighed, ' ||
-    'sum(wer_eligible) AS wer_eligible ' ||
+    'sum(wer_eligible) AS wer_eligible, ' ||
+    'sum(has_aadhar_id) AS child_has_aadhar ' ||
     'FROM ' || quote_ident(_child_health_tablename) || ' ' ||
```

Comment:
may be more clear to call this `_agg_child_health_tablename` but that's pre-existing